src/fips.c

```diff
@@ -159,6 +209,7 @@ fips_init(void)
 		FIPS_mode_set(1);
 	}
 }
+#if 0
```

Comment:
I guess this is my bad.  We should probably remove this code and not just comment it out.

Reply:
Yes, makes sense. Removed.